recipes-extended/xen/files/libxl-domain-state.patch

```diff
@@ -120,7 +120,7 @@ Index: xen-4.6.4/tools/libxl/libxl_utils.c
  
 +int libxl_update_state_direct(libxl_ctx *ctx, libxl_uuid xl_uuid, const char * state)
 +{
-+    char path[48];
++    char path[sizeof("/state/00000000-0000-0000-0000-000000000000/state")];
```

Comment:
Nothing wrong with this as I would expect that the compiler optimizes out the sizeof. So really just a preference/suggestion, but I tend to lean towards giving example in comment with a defined value that provides a sensible name below the comment. I would also do this for uuid array, but I would bet the uuid implementation already provides a defined value.